packages/live-share-inking/src/canvas/InkingCanvas.ts

```diff
@@ -0,0 +1,526 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the Microsoft Live Share SDK License.
+ */
+
+import { TWO_PI, IPointerPoint, IQuad, IPoint, viewportToScreen } from "../core/Geometry";
+import { IStroke } from "../core/Stroke";
+import { IBrush } from "../core/Brush";
```

Comment:
You may be doing this on purpose (which is fine) but just making sure you know you can import all of these from "../core" since you have an index.ts... That's what pressing <tab> to auto-import is going to do.

Reply:
Sure I can do that,

Reply:
Fixed in https://github.com/microsoft/live-share-sdk/commit/18b13a631f0c2833a77b7d5181cb89b36206c512